### PR TITLE
Linking class to ui

### LINKED ISSUE #145: Link data types to UI
Link the `ApplicationItem`, `ProfileItem`, `CompanyItem` to UI.

### PR DESCRIPTION
**Pull request by seanjyjy**
Implemented the linking of the data types to the UI and made a few other changes along the way.

PR overview:

* Link displays(right-hand side) to the data types. `changeDisplay` method in `MainWindow.java` is not yet completed, have to see the implementation of view.

* Added 1 more boolean value to commandResults for `view`.

* Reformatted and edited some magic strings that we all have in common in `getMapping` into `PanelDisplayKeyword.java`.

* Added StubDatas in `tempStubData` for us to see whether the implementation is fine, to be deleted, and also link to model once the implementation to link the data types to UI is ok.

* Some slight UI issue that arises are
    * the words inside the tags are unable to aligned yet,
    * For Company, I am not sure what is the Internship representation is supposed to be on the right display.
    * For Profile, I am not sure if I should put everything inside the right display.
    * It might look a bit empty for the profile.

* The three different screens current UI are below. 
Screenshot 2020-10-10 at 7 05 18 PM
Screenshot 2020-10-10 at 7 05 11 PM
Screenshot 2020-10-10 at 7 05 14 PM

### CODE REVIEW

File: src/main/java/seedu/address/ui/cards/Card.java [hunk 1]
```diff
@@ -129,9 +152,10 @@ public boolean equals(Object other) {
             return false;
         }
 
+        @SuppressWarnings("unchecked")
```
Comment (seanjyjy) [inline src/main/java/seedu/address/ui/cards/Card.java:155]: Not very sure how to resolve this portion. Maybe I should a raw type parameterized instead?

File: src/main/java/seedu/address/ui/cards/ApplicationCard.java [hunk 2]
```diff
@@ -1,74 +1,174 @@
...
     }
 
     /**
      * todo Javadocs
      */
-    private void initializeDate() {
-        Image calendarIcon = new Image(this.getClass().getResourceAsStream(CALENDAR_IMAGE_LINK));
-        calendar.setImage(calendarIcon);
-        date.setText("21/07/2020"); // to be changed later!
+    private void setTags() {
```
Comment (shawn-nyk) [inline src/main/java/seedu/address/ui/cards/ApplicationCard.java:79]: Minor nit, but perhaps we could change this to `setRequirements` in the future? Just to keep all the namings consistent.

File: src/main/java/seedu/address/ui/cards/CompanyCard.java [hunk 3]
```diff
@@ -1,27 +1,75 @@
...
      * todo Javadocs
      */
     private void initializeCompanyCardGui() {
-        // to be edited in the future!
-        initializeHeader(displayedIndex);
-        initializeBody();
-        initializeTags();
+        setId(displayedIndex);
+        setName();
+        setTags();
```
Comment (shawn-nyk) [inline src/main/java/seedu/address/ui/cards/CompanyCard.java:30]: Same thing regarding the tags naming here. Unless I'm misunderstanding the use of the word "tag" in the context of this UI portion of code. Apologies if that's the case!

File: src/main/java/seedu/address/ui/panel/ListPanel.java [hunk 4]
```diff
@@ -1,32 +1,36 @@
 package seedu.address.ui.panel;
 
+
```
Comment (shawn-nyk) [inline src/main/java/seedu/address/ui/panel/ListPanel.java:3]: Perhaps this newline is an extra addition?

**Review by shawn-nyk — APPROVED** [review]
Fantastic work Sean! Just left a few comments but they are very minor so LGTM! Regarding the issues brought up, perhaps we can settle them during subsequent meetings.

File: src/main/java/seedu/address/ui/cards/ApplicationCard.java (hunk 2 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/cards/ApplicationCard.java:79]: yup I think I should change this to `setRequirements`

File: src/main/java/seedu/address/ui/cards/CompanyCard.java (hunk 3 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/cards/CompanyCard.java:30]: yup I think I should change this to `setIndustries`

**Pull request merged**